config/peerpods/podvm/podvm-builder.sh

```diff
@@ -189,7 +224,7 @@ function delete_podvm_image_gallery() {
 }
 
 function display_usage() {
-  echo "Usage: $0 {create|delete [-f]|delete-gallery [-f]}"
+  echo "Usage: $0 {create|delete [-f]|delete-gallery [-g]}"
```

Comment:
```suggestion
  echo "Usage: $0 {create|delete [-f] [-g]|delete-gallery [-f]}"
```

Reply:
Thanks.. Not sure what I was doing when making this change :-(